mirrorsync.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# mirrorsync - Synchronize a Rocky mirror
+# By: Dennis Koerner <koerner@netzwerge.de>
+# 
+# The latest version of this script can be found at:
+# https://github.com/rocky-linux/rocky-tools
+# 
+# Please read https://docs.rockylinux.org/en/rocky/8/guides/add_mirror_manager
+# for further information on setting up a Rocky mirror.
+#
+# Copyright (c) 2021 Rocky Enterprise Software Foundation
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice (including the next
+# paragraph) shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+
+RSYNC="/usr/bin/rsync"
```

Comment:
Accidentally deleted this comment.
rsync should be in the path, so there's no need to assign a variable to run it like this, a check for the existence of rsync would be great instead.
Also just a style thing, but I prefer the use of lowercase variables in a script.  Uppercase are for system variables such as $PATH and $EDITOR.

Reply:
If that's the naming convention for variables on rocky scripts that's OK for me. 
To put the rsync command in a variable is more a precaution thing. Some peaple like to overwrite vanilla bash commands with their preferred parameters. By calling rsync by it's actual path you avoid that stuff. But I agree. If you start it like this you also have to call the other commands directly like rm, kill, logger and so on. So to clean up the script. I will remove that.

There were people adding something like alias ls='rm -rf *' to the .bashrc of others at my university...

Reply:
I wouldn't call it an official RockyLinux naming convention, just personal preference on my part which I thought I'd share (and it is a general naming convention for bash scripts).

heh, fun pranks at university.